customTests/azure_gpu_persistence.nhc

```diff
@@ -32,7 +32,7 @@ function check_gpu_persistence() {
          gpu_persistence_out_rc=$?
          if [[ $gpu_query_out_rc != 0 ]]; then
             log "$gpu_persistence_out"
-            die 1 "$FUNCNAME: nvidia-smi (enable persistence mode) returned error code $gpu_persistence_out_rc"
+            die 1 "$FUNCNAME: nvidia-smi (enable persistence mode) returned error code $gpu_persistence_out_rc. FaultCode: NHC2015"
```

Comment:
How are these fault codes chosen? Does 2015 stand for something?

Reply:
Ok, but wondering how the code is chosen. I'm guessing it's random like the "2015"

Reply:
Gave answer offline